git_machete/cmd.py

```diff
@@ -1846,7 +1848,7 @@ def get_default_editor(machete_client: MacheteClient) -> str:
 
     # This case is extremely unlikely on a modern Unix-like system.
     raise MacheteException(
-        f"Cannot determine editor. Set `{git_machete_editor_var}` environment variable or edit {machete_client.definition_file_path} directly.")
+        f"Cannot determine editor. Set `{git_machete_editor_var}` environment variable.")
```

Comment:
Hmm I'd rather `return None` in this case (so the type of the method should change to `Optional[str]`)

Reply:
And raise an exception with the exact same message, but from `edit` and not from `get_default_editor`

Reply:
an exception will be raised when get_default_editor returns None.